components/form/textarea/src/index.js

```diff
@@ -0,0 +1,66 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+
+const FormTextarea = ({className, cols, label, name, onChange, placeholder, rows, spellCheck, value}) => (
+  <div className={cx('sui-FormTextarea', className)}>
+    {label && <label className='sui-FormTextarea-label'>{label}</label>}
+    <textarea
+      className='sui-FormTextarea-element'
+      cols={cols}
+      name={name}
+      onChange={onChange}
+      placeholder={placeholder}
+      rows={rows}
+      spellCheck={spellCheck}
+      value={value} />
+  </div>
+)
+
+FormTextarea.displayName = 'FormTextarea'
+
+FormTextarea.propTypes = {
+  /**
+   * CSS classname to apply to component's container.
+   */
+  className: PropTypes.string,
+  /**
+   * Specifies the visible width of text area.
+   */
+  cols: PropTypes.number,
+  /**
+   * Title label to show above the text area element.
+   */
+  label: PropTypes.string,
+  /**
+   * Specifies a name for the text area.
+   */
+  name: PropTypes.string.isRequired,
```

Comment:
Why is this required?

Reply:
Same as pointed in the checkbox's PR, I will remove this 👍